docs/resources/group.md

```diff
@@ -20,10 +20,74 @@ resource "opal_group" "opal_group_example" {
   admin_owner_id = opal_owner.security.id
   require_mfa_to_approve = true
   auto_approval = false
-  require_manager_approval = true
 
-  reviewer {
-    id = opal_owner.security.id
+  reviewer_stage {
+    reviewer {
+      id = opal_owner.security.id
+    }
+  }
+}
```

Comment:
I was thinking to make it a little more obvious what the reviewer.id should be:
```tf
  reviewer_stage {
    owner {
      id = opal_owner.security.id
    }
  }
}
```
we use `admin_owner_id` to indicate that the admin has to be an owner, we could follow suit here as well. What do you think?

Reply:
hmm...I think I prefer the current state for 2 reasons:
1. reviewer matches the previous behavior that customers already used
2. we have plans to remove the admin functionality from owners and then possibly rename owners to something more reviewer specific.